node/core/approval-voting/src/import.rs

```diff
@@ -964,9 +991,9 @@ pub(crate) mod tests {
 					keystore: &LocalKeystore::in_memory(),
 				};
 
-				let info = imported_block_info(&mut ctx, env, hash, &header).await.unwrap();
+				let info = imported_block_info(&mut ctx, env, hash, &header).await;
 
-				assert!(info.is_none());
+				assert_matches!(info, Err(ImportedBlockInfoError::BlockFromAncientSession));
```

Comment:
Note: this test might not be testing what it actually wants to test? The test is called `imported_block_info_fails_if_unknown_session`, but apparently it triggers the "block from ancient session" condition.

Reply:
Good catch, thanks!